sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,8 @@
+select "g"."name" as "genreName",
+        count("g".*) as "numberOfAppearances"
+   from "actors"
+   join "castMembers" using ("actorId")
+   join "filmGenre" using ("filmId")
+   join "genres" as "g" using ("genreId")
+   where "actors"."actorId" = 178
```

Comment:
Since you can't guarantee the actor you're looking for will have that id without performing another query (manual or otherwise), please adjust your filter to match exactly who you're looking for:

---

![image](https://user-images.githubusercontent.com/16143771/186543007-96bd0a00-d751-4262-b2ce-7825207bc5e3.png)